stable/traefik/README.md

```diff
@@ -117,6 +117,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `dashboard.statistics.recentErrors` | Number of recent errors to show in the ‘Health’ tab              | None                                      |
 | `service.annotations`           | Annotations for the Traefik Service definition, specified as a map   | None                                      |
 | `service.labels`                | Additional labels for the Traefik Service definition, specified as a map | None                                  |
+| `service.nodePorts`           | Static nodePort allocation for service of type LoadBalancer  | dynamic nodePort allocation                                      |
```

Comment:
This documentation isn't clear. There are actually two separate properties `service.nodePorts.http` and `service.nodePorts.https` that aren't mentioned here. Also, the description and default value are confusing. I guess it means that if not set, it's dynamic, but static if explicitly set? If I'm interpreting that correctly, I believe that should be more explicitly stated in order to limit the guess work required from a reader.

Reply:
yes you are right. will amend